src/runtime/contrib/arm_compute_lib/acl_utils.cc

```diff
@@ -108,6 +111,30 @@ arm_compute::PadStrideInfo ToACLPadStride(const std::vector<std::string>& pad,
                                     arm_compute::DimensionRoundingType::FLOOR);
 }
 
+arm_compute::DataType MakeDataType(const DLDataType& data_type) {
+  if (data_type.code == DLDataTypeCode::kDLFloat && data_type.bits == 32) {
+    return arm_compute::DataType::F32;
+  } else if (data_type.code == DLDataTypeCode::kDLUInt && data_type.bits == 8) {
+    return arm_compute::DataType::QASYMM8;
+  } else if (data_type.code == DLDataTypeCode::kDLInt && data_type.bits == 32) {
+    return arm_compute::DataType::S32;
+  } else {
+    LOG(FATAL) << "Datatype " << data_type << " unsupported by ACL runtime";
+    return arm_compute::DataType::UNKNOWN;
+  }
+}
+
+template <typename T>
+std::vector<T> GetVectorFromDLTensor(const DLTensor* tensor) {
```

Comment:
I feel that we have an existing utility for this. @zhiics.

Reply:
I had a look for something like this in the runtime, although couldn't find anything yet

Reply:
yeah, maybe we have it but I am not sure where it is either. But do we have to flatten the tensor during runtime?

Reply:
Its not so much about flattening the tensor but rather converting it to the type that ACL expects. `arm_compute::QuantizationInfo(std::vector<float> scale, std::vector<int32_t> offset)`